package.json

```diff
@@ -14,6 +14,7 @@
     "storybook": "turbo run start --filter=uikit...",
     "build:storybook": "turbo run build:storybook --filter=uikit...",
     "build": "turbo run build --filter=web...",
+    "build-packages": "cd packages/swap-sdk-core && yarn && yarn build && cd ../swap-sdk && yarn && yarn build && cd ../aptos-swap-sdk && yarn && yarn build && cd ../wagmi && yarn && yarn build && cd ../token-lists && yarn && yarn build && cd ../smart-router && yarn && yarn build && cd ../multicall && yarn && yarn build && cd ../uikit && yarn && cd ../awgmi && yarn && cd ../../",
```

Comment:
https://github.com/pancakeswap/pancake-frontend/blob/73ea15629bb503728a7cd4e8a73157c4bb0c92b1/package.json#L17
latest version has the build package script